ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentsTable.tsx

```diff
@@ -11,11 +11,41 @@ import TooltipTh from 'Components/TooltipTh';
 import DateDistance from 'Components/DateDistance';
 import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
 import { TableUIState } from 'utils/getTableUIState';
+import {
+    generateVisibilityFor,
+    getHiddenColumnCount,
+    ManagedColumns,
+} from 'hooks/useManagedColumns';
 import { getWorkloadEntityPagePath } from '../../utils/searchUtils';
 import SeverityCountLabels from '../../components/SeverityCountLabels';
 import { VulnerabilitySeverityLabel } from '../../types';
 import useVulnerabilityState from '../hooks/useVulnerabilityState';
 
+export const tableId = 'workloadCvesDeploymentOverviewTable';
```

Comment:
Understanding that this is placeholder pending brainstorming by the team:
1. `WorkloadCves` prefix corresponding to folder (as discussed in chat)
2. What are pro and con of  `DeploymentsTable` file name versus`DeploymentOverviewTable` as suffix?
    * One side of coin: reuse previous decisions?
    * Other side of coin: does more specific name suggest what file name would have been in hindsight?
        If yes, to limit changed files, I do not consider file renaming as necessary to preseve pattern.

Reply:
Hmm... the file names of these tables have been on the back burner for a while. If we go with the folder prefix + file name combination, I'd be in favor of just doing the rename now so long as it doesn't result in a large enough change that git detects it as a new file. If so, I could do the rename in a follow up.

Reply:
I went ahead with this change and the file renaming. Although we will still have this on the agenda for the team to collab on, this is an improvement over what was previously in the PR.